draft-ietf-cdni-uri-signing.xml

```diff
@@ -1262,6 +1263,37 @@ http://*/folder/content-83112371/quality_*/segment????.mp4
         number for this document.]</t>
 
       </section>
+
+      <section anchor="sec.IANA.cdnistt"
+               title="CDNI URI Signing Signed Token Transport">
+        <t>The IANA is requested to create a new "CDNI URI Signing
+        Signed Token Transport" subregistry in the "Content
+        Delivery Networks Interconnection (CDNI) Parameters" registry.
+        The "CDNI URI Signing Signed Token Transport"
+        namespace defines the valid values
+        that may be included in a URI Signing token.  Additions to the
```

Comment:
Sorry I wasn't clear enough.  The title of the registry is great. I just pointed out that the term "URI Signing token" at the end of the sentence is not a defined term in the specification. Although I do understand what it is meant, maybe not a normarl reader. We could change "URI Signing token" to "Signed Token" which is used throughout the specification. What do you think ?